test/grpc/BUILD.bazel

```diff
@@ -12,6 +12,8 @@ cc_binary(
         "death-client.cc",
     ],
     copts = copts(),
+    # Use a faster implementation of malloc (and show that tests pass with it).
+    malloc = "@com_github_jemalloc_jemalloc//:jemalloc",
```

Comment:
I'm ok with not using `select()` in `test/grpc/BUILD.bazel` since we don't build these rules on Windows anyway.